STAR-Delegation/draft-ietf-acme-star-delegation.md

```diff
@@ -528,11 +528,11 @@ ACME Delegation process:
   * Similarly, the `star-certificate` URL MUST be copied as-is.
   * The `finalize` URL is rewritten, so that the `finalize` request will be
     made to the proxy.
-  * The `Location` header must be rewritten.
+  * The `Location` header MUST be rewritten to point to an `Order` object on the proxy.
 * Finalize request:
   * The CSR MUST be copied as-is.
 * Finalize response:
-  * Both the `Location` header and the `finalize` URLs are rewritten.
+  * Both the `Location` header and the `finalize` URLs are rewritten as for Get Order.
```

Comment:
Just a thought: should "Get Order" be "POST-as-GET order" instead (as per 8555)?

Reply:
No, IMO this is a logical name of the request, not the actual HTTP method that's being used.